frontend/beCompliant/src/components/table/pagination/PaginationRelativeButtons.tsx

```diff
@@ -0,0 +1,106 @@
+import { Updater } from '@tanstack/react-table';
+import React from 'react';
+import { PaginationActionButton } from './PaginationActionButton';
+import { Center, Flex, Icon } from '@kvib/react';
+
+interface Props {
+  numberOfPages: number;
+  currentIndex: number;
+  setIndex: (updater: Updater<number>) => void;
+}
+
+export function PaginationRelativeButtons({
+  numberOfPages,
+  currentIndex,
+  setIndex,
+}: Props) {
+  if (numberOfPages <= 2) {
+    return <></>;
+  }
+  // If less then 7 total pages display all pages without ...
+  if (numberOfPages <= 7) {
+    return [...Array(numberOfPages - 2)].map((_, index) => {
+      return (
+        <PaginationActionButton
+          ariaLabel={`Gå til side ${index + 2}`}
```

Comment:
Kanskje dette hadde fungert
```suggestion
          ariaLabel={`Gå til side ${numberOfPages + 2}`}
```